hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -297,6 +341,23 @@ def save_slide_node_figures(
                     extra_slide_dict,
                 )
 
+    def get_metadata(self, results: ResultsType) -> Optional[List[Any]]:
+        """
+        Get metadata of outputs (validation or test) from slides dataset to stratify plots (e.g PR curve, ROC curve).
+        """
+        if self.slides_dataset is not None and self.stratify_plots_by is not None:
+            slides_df = self.slides_dataset.dataset_df
```

Comment:
nitpick: maybe this function is easier on the eye if you invert the conditions, like:
```
if self.slides_dataset is None or self.stratify_by is None:
    return None
(then do the rest of what you have in the if-clause now
```

Reply:
thanks, done now